platform/public_api/apps/index.json

```diff
@@ -0,0 +1,744 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Apps Api",
+    "version": "2.0"
+  },
+  "tags": [
+    {
+      "name": "Apps v2"
+    }
+  ],
+  "definitions": {
+    "App_Name": {
+      "type": "string",
+      "description": "Name of an app",
+      "example": "AppName"
+    },
+    "App_Urls": {
+      "description": "Urls of an app",
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "example": [
+        "example.piwik.pro"
+      ]
+    },
+    "App_Timezone": {
+      "description": "Timezone for an app",
+      "type": "string",
+      "example": "UTC"
+    },
+    "App_Currency": {
+      "description": "Currency for an app",
+      "type": "string",
+      "example": "USD"
+    },
+    "App_AddedAt": {
+      "description": "When app was added",
+      "type": "string",
+      "example": 1506570461
+    },
+    "JsonApi_Id": {
+      "type": "string",
+      "format": "uuid",
+      "description": "UUIDv4 string",
```

Comment:
Description specifies format here instead of explaining meaning of the param.

Reply:
here, context is always App so now its `App ID`